### DISCUSSION TITLE
EAF app frames stay on top of all other applications

### DISCUSSION THREAD

**Discussion by Inc0n** [emacs-eaf/emacs-application-framework#981] [Q&A] [answered]
When switching applications, such as firefox or emacs, if an EAF app is currently active (showing in one of the emacs window), its frame will be showing on top of all other applications. How can I fix this?

I am on emacs28, using Mac, git pulled this repo just yesterday.

**Comment by manateelazycat** [comment, marked as answer]
Only linux and windows can follow Emacs window stack, macOS has limit that EAF window will stay on top of all other application.

Did you install mac-app-frontmost ?

mac-app-frontmost will hide EAF window after emacs inactive.

**Reply by Inc0n** [reply]
Hi, yes `mac-app-frontmost` has been installed with `pip`, could you inform me how I can proceed with its configuration? 
Thank you

**Reply by Inc0n** [reply]
I did some more digging, as it turns out, I did not configure PATH properly to allow the binary of `mac-app-frontmost` to be found. I fixed it by add `$HOME/Library/Python/3.9/bin/` to PATH
